### DISCUSSION TITLE
Documentation on publicRuntimeConfig seems lackfull.

### DISCUSSION THREAD

**Discussion by AndersNielsen85** [nuxt/nuxt#18306] [Questions]
Documentation states you can use
```
export default {
  publicRuntimeConfig: {
    baseURL: process.env.BASE_URL || 'https://nuxtjs.org'
  }
}
```
But the corresponding TS seems to match the following:
```
publicRuntimeConfig: {
    app: {
       [key: string]: any
    },
  },

```

Using the boilerplate from `npm nuxi init`
I end up with this to make it work
```
export default defineNuxtConfig({
  publicRuntimeConfig: {
    app: {
```

The documentation says it will be accessible using $config (this works for the template).
In script setup I am able to access it using
`const { app } = useRuntimeConfig();`

If the 'app' is intentional the documentation should be updated to include the 'app' (and maybe how to access in script).

**Comment by danielroe** [comment]
The types issue will be fixed in https://github.com/nuxt/framework/pull/1094. Until we update the docs, see https://github.com/nuxt/framework/discussions/1000#discussioncomment-1483083.
